about/contributing_datasets.md

```diff
@@ -0,0 +1,288 @@
+observationAbout: dcid:geoId/04
+observationDate: "2020-03-04"
+value: 24
+```
+
+For more information on the terms [`StatisticalVariable`](https://browser.datacommons.org/kg?dcid=StatisticalVariable) and [`StatVarObservation`](https://browser.datacommons.org/kg?dcid=StatVarObservation), see ["Intro to `StatisticalVariable` and `StatVarObservation`"](https://github.com/datacommonsorg/data/tree/master/docs/representing_statistics.md).
+
+### Roadmap
+
+Importing data requires three main steps:
+
+1. Defining new `StatisticalVariable` nodes.
+2. Cleaning the CSV files.
+3. Defining a mapping from CSV to MCF (aka [Template MCF](https://github.com/datacommonsorg/data/tree/master/docs/mcf_format.md#template-mcf)).
+
+This tutorial explains each step in detail and demonstrates where you can submit artifacts for Data Commons review.
+
+## Step 1: Define new `StatisticalVariable` nodes.
+
+**Note:** If the dataset does not introduce new variables, skip this step and refer to existing `StatisticalVariable` identifiers.
+
+### Required triples
+
+At a minimum, each `StatisticalVariable` needs a `populationType` and a curated, human readable identifier. Here's an example:
+
+```
+Node: dcid:Count_MedicalTest_COVID_19_Pending
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalTest
+```
+
+In this example, "`Count_MedicalTest_COVID_19_Pending`" is the curated identifier. Data Commons recommends making these IDs descriptive of the stat. For example, if your data is specifically for median income among females age 12 and over, you could assign "MedianIncome_Person_Female_12YearsAndOlder".
+Here "`MedicalTest`" is the common type of the group being observed. E.g. "Person" and "Household" if measuring median income of people and households.
+
+### Optional triples
+
+The `statType` and `measured_property` fields can be updated later. If they are not provided, Data Commons will assign `measuredValue` as the `statType` and use the `dcid` as a placeholder `measured_property`. Here is an example that includes these optional triples:
+
+```
+Node: dcid:Count_MedicalTest_COVID_19_Pending
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+```
+
+Other common `statType`s include `medianValue`, `meanValue` and `minValue`. Other common `measuredProperty`s include `count`, `incrementalCount`, and `age`.
+
+### Remaining triples
+Data Commons will take care of filling in the "constraining properties". These properties typically express information that you have already encoded in the `StatisticalVariable`'s `dcid`. In this example, the `dcid` "Count_MedicalTest_COVID_19_Pending" is split into `medicalCondition: dcs:COVID_19` and `testResult: dcs:Pending`. Here is an example final MCF:
+
+```
+Node: dcid:Count_MedicalTest_COVID_19_Pending
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+medicalCondition:	dcs:COVID_19
```

Comment:
These should be single space separated (not sure if that is a tab separating the value, or many spaces).

Reply:
I...think this should be resolved? I checked the raw version of this document and I didn't see anything.